Entities/Arm.cpp

```diff
@@ -441,7 +441,7 @@ void Arm::UpdateArmFrame() {
 void Arm::Draw(BITMAP *pTargetBitmap, const Vector &targetPos, DrawMode mode, bool onlyPhysical) const {
     Attachable::Draw(pTargetBitmap, targetPos, mode, onlyPhysical);
 
-    if (!onlyPhysical && (mode == g_DrawColor || mode == g_DrawWhite || mode == g_DrawTrans) && (!m_Parent || m_pHeldMO || (!m_pHeldMO && !m_DidReach))) {
+    if (!onlyPhysical && (mode == g_DrawColor || mode == g_DrawWhite || mode == g_DrawTrans) && (m_pHeldMO || !m_DidReach) && m_Parent) {
```

Comment:
Please undo this - I've (hopefully) fixed the actual problem with Arms so we can draw hands on detached arms now, which your change doesn't allow.
Btw, you can find the original logic for it here, might help you better understand what's happening/supposed to be happening:
https://github.com/DataRealms/CCOSS/blob/master/Entities/Arm.cpp